index.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Mobile-Portfolio</title>
+</head>
+<body>
+    <header>
+        <nav class="nav-bar">
+            <a href="#">My Logo</a>
+            <img src="/images/Union.png" alt="humburger">
```

Comment:
- Kindly fix your path for the hamburger icon, there's a dot `.` missing at the beginning I believe 💯 

![image](https://user-images.githubusercontent.com/93630700/227223547-c7f4ce89-5b95-4d14-98fa-887622a5fc52.png)